src/commands/update/handle_update_command.py

```diff
@@ -1,13 +1,19 @@
 from os import getcwd, listdir, path
 
 from src.commands.update.update_package import update_package
+from src.utils import retrieve_real_package_name
 
 
 def handle_update_command(args) -> None:
     assert args.command == "update"
 
+    root_repo_dir = getcwd()
```

Comment:
I will have a solution for getting those (pkg root, lib root) in a separate PR - can you make me a task and leave a todo here to integrate it?

Reply:
https://github.com/software-mansion/protostar/issues/55